src/RenderingSystem.cpp

```diff
@@ -68,6 +60,44 @@ void RenderingSystem::handle_key_press(const Event& e) {
 
 }
 
+void RenderingSystem::handle_add_example_ship(const Event& e) {
+    // Load game object parameters
+    int object_id = e.get_value<int>("object_id", -1);
+    assert(object_id != -1);
+
+    int x = e.get_value<int>("pos_x", -999);
+    assert(x != -999);
+
+    int y = e.get_value<int>("pos_y", -999);
+    assert(y != -999);
+
+    int z = e.get_value<int>("pos_z", -999);
```

Comment:
Will we need values more granular than ints?

Reply:
Not sure I understand your question. We'll prob want to pass position as a glm::vec3, etc.